cni/network/plugin/main.go

```diff
@@ -83,11 +83,30 @@ func main() {
 
 	netPlugin.SetReportManager(reportManager)
 
+	defer func() {
+		errUninit := netPlugin.Plugin.UninitializeKeyValueStore()
+		if errUninit != nil {
+			log.Printf("Failed to uninitialize key-value store of network plugin, err:%v.\n", err)
+		}
+
+		if recover() != nil {
+			os.Exit(1)
+		}
+	}()
+
+	err = netPlugin.Plugin.InitializeKeyValueStore(&config)
+
```

Comment:
remove blank and you can check for err in same line as i said below

Reply:
Done. Changed to this stype in other places as well